### PR TITLE
ZEN-31701 Update MariaDB to more current version

### PR DESCRIPTION
**Pull request by romanbakaleyko**

### CODE REVIEW

File: script/eval.go [hunk 1]
```diff
@@ -60,16 +60,39 @@ func evalSnapshot(r *runner, n node) error {
 func evalUSE(r *runner, n node) error {
 
 	imageName := n.args[0]
-	replaceImgs := make([]string, len(n.args)-1)
-	//if len(n.args) > 1 && n.args[1] != "" {
+	replaceImgs := make([]string, 0)
```
Comment (jplouis) [inline script/eval.go]: I know this code is poorly commented, but add a comment to the effect of:

If "service" is in the SVC_USE command the image will be replaced only on the service (and its children) listed after the "service" keyword.  eg.  `SVC_USE [PREVIOUS_IMAGE_NAMES] service <SVC_PATH>`

File: script/eval.go [hunk 2]
```diff
@@ -60,16 +60,39 @@ func evalSnapshot(r *runner, n node) error {
 func evalUSE(r *runner, n node) error {
```
Comment (jplouis) [inline script/eval.go]: make sure to add unit test for the new usages of SVC_USE

Comment (romanbakaleyko) [reply script/eval.go]: in the script package I don't see tests for eval.go, regarding functionality in parser.go and nodes.go I am not sure what else I can add there. For ServiceUse facade method I don't see tests as well.

File: script/eval.go (hunk 1 above)
Comment (jplouis) [reply script/eval.go]: Please add comments on the expected format of the updated command

File: script/eval.go [hunk 3]
```diff
@@ -57,21 +57,56 @@ func evalSnapshot(r *runner, n node) error {
...
+
+	tenantID, found := r.env["TENANT_ID"]
+	if !found {
+		return fmt.Errorf("no service tenant id specified for %s", USE)
+	}
+
 	if len(n.args) > 1 {
-		replaceImgs = n.args[1:]
+		for i, arg := range n.args[1:] {
+			if arg == "service" {
```
Comment (jplouis) [inline script/eval.go]: I forgot to save this comment yesterday.  What happens when the keyword "service" is in the wrong location?\
eg. SVC_USE <new image> service <image to replace>  Zenoss.resmgr/Infrastructure/mariadb-model

Would like a test case for that.

File: script/eval.go (hunk 2 above)
Comment (jplouis) [reply script/eval.go]: There are tests in parser_test.go. https://github.com/control-center/serviced/blob/develop/script/parser_test.go#L36

It uses a test file to make sure the parsing goes as expected. There are also negative test cases at https://github.com/control-center/serviced/blob/develop/script/parser_test.go#L119

File: script/eval.go (hunk 3 above)
Comment (romanbakaleyko) [reply script/eval.go]: updated parser,  added check whether we have image to replace in the command and check if we have path_to_service specified.

File: script/eval.go (hunk 2 above)
Comment (romanbakaleyko) [reply script/eval.go]: added tests

**Pull request merged**